docs/06-troubleshooting/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Troubleshooting
+description:
+  This page contains the most common errors you may encounter, along with
+  information that will help to solve them quickly.
+---
+
+<p>{frontMatter.description}</p>
+
+## CSS can disappear in development
+
```

Comment:
**nitpick:** it only appears since 3.4, and may be fixed in the future (even if unrealistic for this one)

**suggestion:** add a `<SinceVersion>` tag under the title